templates/supertest/put/put.handlebars

```diff
@@ -1,4 +1,4 @@
-  it('should respond with {{length description}}', function(done) {
+  it('should respond with {{length description}}{{#if inputs}}, {{inputs.body.message}}{{/if}}', function(done) {
```

Comment:
that was exactly what I meant and understood already :-). I was inquiring about the new `{{if}}` statement in there and the `inputs.body.message`, which seems user supplied.

But lets wait for @mm-gmbd